pages/common/git-commit.md

```diff
@@ -11,7 +11,7 @@
 
 `git commit -a -m {{message}}`
 
-- Replace the last commit with currently staged changes:
+- Update the last commit by adding the currently staged changes, changing commit's hash:
```

Comment:
This looks better, but there's a missing article here. Perhaps this would read better?
```suggestion
- Update the last commit by adding the currently staged changes, changing the commit's hash:
```

Reply:
@sbrl it should be good now